content/3-getting-started/default.txt

```diff
@@ -82,7 +82,7 @@ SVG.js also works outside of the HTML DOM, inside an SVG document for example:
   <script type="text/javascript" xlink:href="svg.min.js"></script>
   <script type="text/javascript">
     <![CDATA[
-      var draw = SVG('drawing')
+      var draw = SVG().addTo('drawing')
```

Comment:
```suggestion
      var draw = SVG().addTo('#drawing')
```

Reply:
This one is not correct. When you use svg.js in an SVG document you dont want to create a new SVG in the existing one. You mostly only want to get the reference of the SVG itself. 
Thats why you would use the syntax: `SVG('#drawing')` which gives you back the svg.js object for the root svg

Reply:
It's so unreliable you have to learn to appreciate the fact that you just fucking follow the document and report an error